app/ide-desktop/lib/dashboard/src/components/dashboard/column/LabelsColumn.tsx

```diff
@@ -141,10 +141,11 @@ export default function LabelsColumn(props: column.AssetColumnProps) {
         ))}
       {managesThisAsset && (
         <ariaComponents.Button
-          size="custom"
-          variant="custom"
           ref={plusButtonRef}
-          className="shrink-0 rounded-full opacity-0 group-hover:opacity-100 focus-visible:opacity-100"
+          size="icon"
+          variant="outline"
+          className="opacity-0 group-hover:opacity-100 focus-visible:opacity-100"
```

Comment:
AFAIR `group` here relates to this button, doesn't it?

Reply:
if you mean the `group` in the ancestor `div`, then yes

Reply:
Button creates its own group: https://github.com/enso-org/enso/blob/ccfeac1c02b3f6637f1e66650d712afb1f175af1/app/ide-desktop/lib/dashboard/src/components/AriaComponents/Button/Button.tsx#L69

I'm not sure how tw will resolve `group-*` selector in this situation.

Reply:
ah, good to know. i think it's not a problem as `.group` matches any group:

![image](https://github.com/enso-org/enso/assets/4046547/f693a23a-68f2-411e-aacc-baf082e523cf)

and all we care about is that the button is visible when the `group` is hovered/contains focus.